src/main/typescript/components/form/FoldableCard.tsx

```diff
@@ -52,12 +52,10 @@ class FoldableCard extends Component<FoldableCardProps & FoldableCardInputProps>
     render() {
         const { title, required, open, children } = this.props
 
+        const x = `${open ? "" : "closed"} card mb-3`.trim()
```

Comment:
what's this?

Reply:
good catch! old code for an experiment. Removed it in 69155c3